vendor/sigs.k8s.io/cloud-provider-azure/pkg/provider/azure_controller_common.go

```diff
@@ -476,8 +476,8 @@ func (c *controllerCommon) UpdateVM(ctx context.Context, nodeName types.NodeName
 	return vmset.UpdateVM(ctx, nodeName)
 }
 
-// getNodeDataDisks invokes vmSet interfaces to get data disks for the node.
-func (c *controllerCommon) getNodeDataDisks(nodeName types.NodeName, crt azcache.AzureCacheReadType) ([]compute.DataDisk, *string, error) {
+// GetNodeDataDisks invokes vmSet interfaces to get data disks for the node.
```

Comment:
Please submit a PR with following changes in `cloud-provider-azure` to https://github.com/edreed/cloud-provider-azure

Reply:
Done. I submitted the PR for it: https://github.com/edreed/cloud-provider-azure/pull/172